web/src/components/Table/ManifestListSize.tsx

```diff
@@ -0,0 +1,38 @@
+import {Skeleton} from '@patternfly/react-core';
+import prettyBytes from 'pretty-bytes';
+import {Manifest} from 'src/resources/TagResource';
+import {useRecoilValueLoadable} from 'recoil';
+import {childManifestSizeState} from 'src/atoms/TagListState';
+
+export default function ManifestListSize(props: ManifestListSizeProps) {
+  const childManifestSizes = props.manifests.map((manifest) =>
+    useRecoilValueLoadable(childManifestSizeState(manifest.digest)),
```

Comment:
do we have to use `useRecoilValueLoadable` instead of `useRecoilValue` here? IIUC `useRecoilValueLoadable` is for asynchronous operations happening within the atom.

Reply:
I think I misunderstood this then, you are right, there is no async operation inside the atom, it just gets mutated asynchronously. Will refactor.